src/client/voice/VoiceConnection.js

```diff
@@ -447,7 +447,7 @@ class VoiceConnection extends EventEmitter {
      */
     if (this.status === VoiceStatus.CONNECTED) {
       this.emit('speaking', user, speaking);
-      if (!speaking) {
+      if (!speaking.has('SPEAKING')) {
```

Comment:
The lib convention is to use the static constants instead of string literals. So:

```js
      if (!speaking.has(Speaking.FLAGS.SPEAKING)) {
```

Reply:
That makes more sense! I'm very new to node.js, and figured there must be something like that that I could use :) 
I'll look at that now and test before updating here